docs/ops/movement/Split_1.md

```diff
@@ -6,39 +6,46 @@
 
 **Short description**: *Split* operation splits an input tensor into pieces of the same length along some axis.
 
-**Attributes**
+**Detailed Description**
 
-* *num_splits*
+*Split* operation splits a given input tensor `data` into chunks of the same length along a scalar `axis`. It produces multiple output tensors based on *num_splits* attribute.
+The i-th output tensor shape is equal to the input tensor `data` shape, except for dimension along `axis` which is `data.shape[axis]/num_splits`.
 
-  * **Description**: it specifies the number of outputs into which the initial "*data*" tensor will be split along *"axis"*
-  * **Range of values**: a positive integer less than or equal to the size of the dimension being split over
-  * **Type**: any integer type
-  * **Default value**: None
-  * **Required**: *Yes*
+\f[
+shape\_output\_tensor = [data.shape[0], data.shape[1], \dotsc , split\_lengths[axis], \dotsc data.shape[D-1]]
+\f]
 
-**Inputs**
+Where D is the rank of input tensor `data` and `split_lengths[axis]` represents:
 
-* **1**: "data" - A tensor of type T1. **Required.**
+\f[
+split\_lengths[axis] = data.shape[axis] / num\_splits
+\f]
 
-* **2**: "axis" - axis along *"data"* to split. A scalar of type T2 with value from range `-rank(data) .. rank(data)-1`. Negative values address dimensions from the end. **Required.**
+The sum of `split_lengths` along output shapes must match `data.shape[axis]`.
```

Comment:
I would recommend this sentence to be replaced with something like "The axis being split must be evenly divided by *num_splits*" and remove variable "split_lengths" at all and in the formulae for "shape_output_tensor" just use "data.shape[axis] / num_splits" instead of variable  "split_lengths"

Reply:
Done